subsys/bluetooth/host/classic/avctp_internal.h

```diff
@@ -0,0 +1,70 @@
+/** @file
+ *  @brief Audio Video Control Transport Protocol internal header.
+ */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright (C) 2024 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define BT_L2CAP_PSM_AVCTP 0x0017
+
+typedef enum __packed {
+	BT_AVCTP_IPID_NONE = 0b0,
+	BT_AVCTP_IPID_INVALID = 0b1,
+} bt_avctp_ipid_t;
+
+typedef enum __packed {
+	BT_AVCTP_CMD = 0b0,
+	BT_AVCTP_RESPONSE = 0b1,
+} bt_avctp_cr_t;
+
+typedef enum __packed {
+	BT_AVCTP_PKT_TYPE_SINGLE = 0b00,
+} bt_avctp_pkt_typ_t;
```

Comment:
s/typ/type/ (the enum value uses "TYPE" so you should be consistent about this)

Reply:
Done.